Seedshop/ShopSlot.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using Godot;
+using untitledplantgame.Common;
+
+public partial class ShopSlot : Panel
+{
+	private readonly Logger _logger = new("Seedshop");
+
+	PackedScene tooltip = GD.Load<PackedScene>("res://Seedshop/Tooltip.tscn");
+
+	public override void _Ready()
+	{
+		Connect("mouse_entered", new Callable(this, nameof(OnMouseEntered)));
+		Connect("mouse_exited", new Callable(this, nameof(OnMouseExited)));
+	}
+
+	public override void _Process(double delta) { }
+
+	private async void OnMouseEntered()
+	{
+		_logger.Debug("Mouse entered");
+		var tooltipInstance = (Tooltip)tooltip.Instantiate();
+		tooltipInstance.origin = "Seedshop";
+		tooltipInstance.slot = GetNode<Label>("Panel/Name").Text;
+		_logger.Debug("Slot name: " + tooltipInstance.slot);
+		_logger.Debug("Tooltip origin: " + tooltipInstance.origin);
+
+		float x = GlobalPosition.X + GetRect().Size.X;
+		float y = GlobalPosition.Y;
+		// float tempx = GetNode<ColorRect>("/root/TestInventoryScene/Seedshop/ColorRect").GlobalPosition.X;
+		// _logger.Debug("Scale: " + tempx);
+		Vector2I position = (Vector2I)new Vector2(x, y);
+		tooltipInstance.Position = position;
+		tooltipInstance.Transparent = true;
+
+		AddChild(tooltipInstance);
+		_logger.Debug("Tooltip valid: " + tooltipInstance.valid);
+
+		_logger.Debug("has node: " + HasNode("Tooltip"));
+		await ToSignal(GetTree().CreateTimer(0.5f), "timeout");
```

Comment:
Are tooltips being created every time a mouse enters a Slot?

This looks a little heavier than it needs to be.
Since each itemslot has its own tooltip anyways, extract instantiation to `_Ready`.
Leaving only update and visibility logic in this function.

(Assuming the timer is a bandaid to fix times when the tooltip has not properly ready'd up yet)
Use some ready event instead to avoid possible concurrency problems. Or move creation to ready, and avoid the problem all together.

Reply:
Auch im Ready erstellt das so viele tooltips wie es shopslots gibt. Nachteil dabei ist das die Tooltips nach de ersten Anzeigen nicht mehr wieder kommen, weil die gefreed werden. 
Sehe das eher besser die onMouseEnter zu instanzieren und zu löschen sobald maus weg ist anstatt die gesamt im speicher zu halten